geyser-plugin-interface/src/geyser_plugin_interface.rs

```diff
@@ -158,9 +162,31 @@ pub enum ReplicaTransactionInfoVersions<'a> {
     V0_0_2(&'a ReplicaTransactionInfoV2<'a>),
 }
 
+#[derive(Clone, Debug)]
+pub struct ReplicaEntryInfo<'a> {
+    /// The slot number of the block containing this Entry
+    pub slot: Slot,
+    /// The Entry's index in the block
+    pub index: usize,
+    /// The number of hashes since the previous Entry
+    pub num_hashes: u64,
+    /// The Entry's SHA-256 hash, generated from the previous Entry's hash with
+    /// `solana_entry::entry::next_hash()`
+    pub hash: &'a [u8],
+    /// The number of executed transactions in the Entry
+    pub executed_transaction_count: u64,
```

Comment:
I’m curious why the “executed_” prefix was added here, as Transaction execution is not really a concept at the Entry level.

Reply:
Tbh, I was mirroring the fields in the block notification. I agree that execution is not something that an Entry is really aware of, but on the other hand, it's not actually incorrect, as only executed transactions get recorded.
I don't feel strongly; happy to remove `executed` here if you think it's more clear

Reply:
Na im good. Thanks